sentry_streams/tests/test_mypy_integration.py

```diff
@@ -120,7 +120,7 @@ def filter_events(msg: Message[IngestMetric]) -> bool:
 def create_correct_pipeline():
     return (
         streaming_source("input", "test-stream")
-        .apply(Parser("parser", msg_type=TestMessage))              # bytes -> TestMessage
+        .apply(Parser[TestMessage]("parser"))              # bytes -> TestMessage
         .apply(Filter("filter", function=TestFilterCorrect()))      # TestMessage -> TestMessage
```

Comment:
```suggestion
        .apply(Parser[TestMessage]("parser"))                         # bytes -> TestMessage
        .apply(Filter("filter", function=TestFilterCorrect()))      # TestMessage -> TestMessage
```
tiny nit: we should keep the indenting in the test comments here